runtime/tchannel_client.go

```diff
@@ -219,8 +219,14 @@ func (c *tchannelOutboundCall) finish(err error) {
 }
 
 func (c *tchannelOutboundCall) logFields() []zapcore.Field {
+	var hostPort string
```

Comment:
or `fields = append(fields, hostPort` if call is not nil, less noise in log I think? cc @ravirajj

Reply:
i that case its a broader question about convention. Do we want to always log an empty thing for that type in case of nil or do we always want to skip nil fields in the log altogether? Either way is fine but better to be consistent one way or another.

Reply:
discussed offline, planning to not log on nil.